control/ipmitool.spec.in

```diff
@@ -64,6 +64,626 @@ fi
 
 
 %changelog
+* Sat Oct 8 2016 <pere@hungry.com>  1.8.18-1
```

Comment:
<pere@hungry.com> is Petter Reinholdtsen. See file AUTHORS.
On branch master, the last entry in ./control/ipmitool.spec.in has "* Wed Feb 25 2009 <pere@hungry.com>  1.8.11-1"
I've just updated this file to have it at the same level than ./Changelog and also used pere@hungry.com for each added version entry. I need an email for each version in this file; which one should I use instead ?
The Changelog file ends at "version 1.8.18 2016-10-08". Yes, clearly it is not up-to-date. Is it still maintained ? Should it be updated from git log in some way ?

Reply:
I believe it only makes sene to update the Changelog file as well as changelog in the spec when code is freezed and a new release is being prepared. Otherwise each commit would require updating them.

Reply:
Which email address do you recommend me to put for the entries I've added to the changelog in the spec file ?

Reply:
OK, I've put my email for the releases I've added to the changelog.